src/utils/git/getLatestRelease.ts

```diff
@@ -1,14 +1,18 @@
 import * as semver from 'semver'
 import { getTag, TagPointer } from './getTag'
 
+function isValid(tag: string) {
+  return semver.valid(tag);
+}
+
 export function byReleaseVersion(left: string, right: string): number {
   return semver.rcompare(left, right)
 }
 
 export async function getLatestRelease(
   tags: string[],
 ): Promise<TagPointer | undefined> {
-  const allTags = tags.sort(byReleaseVersion)
+  const allTags = tags.filter(isValid).sort(byReleaseVersion)
```

Comment:
I'd drop the `isValid` shorthand because it doesn't bring us much value apart from being shorter. Let's go with

```suggestion
  const allTags = tags.filter((tag) => {
    return sermver.isValid(tag)
  }).sort(byReleaseVersion)
```

Reply:
I agree with this, just guessed from `byReleaseVersion` (which isn't a lot of context) you'd prefer it the other way!